### PR TITLE
fix: makes crop health client work

### PR DESCRIPTION
**Pull request by daniejoh**
The generated api client did not accept the contentType that the crop health api returns. This fixes that.

### CODE REVIEW

File: src/main/java/io/openepi/crop_health/ApiClient.java [hunk 1]
```diff
@@ -905,7 +905,7 @@ public <T> T deserialize(Response response, Type returnType) throws ApiException
             // ensuring a default content type
             contentType = "application/json";
         }
-        if (isJsonMime(contentType)) {
+        if (isJsonMime(contentType) || contentType.startsWith("text/plain")) {
```
Comment (kstigen) [inline src/main/java/io/openepi/crop_health/ApiClient.java:908]: Why is this needed? Are we returning the wrong content-type from the CropHealt API?

Comment (daniejoh) [reply src/main/java/io/openepi/crop_health/ApiClient.java:908]: Did not think to question that 😅 But yes, I think it returns the wrong type:
image

Comment (kstigen) [reply src/main/java/io/openepi/crop_health/ApiClient.java:908]: Yes, we should probably fix it in the crop-health API instead of doing this here.
